src/typescript/frontend/src/components/geoblocking/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+
+import Text from "components/text";
+
+const Item: React.FC = () => {
+  return (
+    <div className="flex items-center justify-center min-w-[fit-content] gap-[16px]">
+      <Text
+        className="h-[2.4rem] whitespace-nowrap"
+        lineHeight="2.4rem"
+        textScale="display6"
+        color="black"
+        textAlign="center"
+        fontWeight="600"
+      >
+        {"blah ".repeat(40).toUpperCase()}
+      </Text>
+    </div>
+  );
+};
+
+export const GeoblockedBanner: React.FC = () => {
+  const items: React.ReactNode[] = [];
+  for (let i = 0; i < 4; i++) {
+    items.push(<Item key={i} />);
+  }
+  return (
+    <div className="w-full bg-pink">
+      <div className="overflow-hidden w-full flex-row">
+        <div className="flex">
+          <div className="flex gap-[16px] animate-banCarousel">{items}</div>
+        </div>
+      </div>
+    </div>
+  );
+};
```

Comment:
```suggestion
export const GeoblockedBanner: React.FC = () => {

  return (
    <div className="w-full bg-pink">
      <div className="overflow-hidden w-full flex-row">
        <div className="flex">
          <div className="flex gap-[16px] animate-banCarousel">
            {Array.from({ length: 4 }).map((_, i) => (
              <Item key={i} />
            ))}
          </div>
        </div>
      </div>
    </div>
  );
};
```

Can always do something like this as well, personally I try to avoid creating React/JSX elements in the render function body and prefer to put them in the return function instead, since it makes it a little more clear what the actual DOM structure will be when rendered

But really it's fine either way